stl.go

```diff
@@ -209,20 +209,32 @@ func ReadFromSTL(i io.Reader) (o *Subtitles, err error) {
 
 		// Parse TTI block
 		var t = parseTTIBlock(b, g.framerate)
-
+		
 		if t.extensionBlockNumber != extensionBlockNumberReservedUserData {
 
+			var starttime=g.timecodeStartOfProgramme
+			if starttime>t.timecodeIn{
+				starttime=0
+			}
 			// Create item
 			var i = &Item{
-				EndAt:   t.timecodeOut - g.timecodeStartOfProgramme,
-				StartAt: t.timecodeIn - g.timecodeStartOfProgramme,
+				EndAt:   t.timecodeOut - starttime,
+				StartAt: t.timecodeIn - starttime,
 			}
 
 			// Loop through rows
 			for _, text := range bytes.Split(t.text, []byte{0x8a}) {
-				parseTeletextRow(i, ch, func() styler { return newSTLStyler() }, text)
+				parseTeletextRow(i, ch, func() styler { return newSTLStyler() }, text)				
 			}
 
+			if i.InlineStyle!=nil{
+				fmt.Printf("InlineStyle not NULL\n")
+			}else{
+				i.InlineStyle=&StyleAttributes{}
+			}
+			i.InlineStyle.TeletextVerticalPostion=t.verticalPosition
```

Comment:
I've the feeling those new attributes `TeletextVerticalPostion` and `TeletextJustificationCode` are specific to `stl` format. If this is the case, I'd rather call them `STLVerticalPostion` and `STLJustificationCode`

Also, there's a type in `TeletextVerticalPostion`, you forgot the `i` in `Position`